### PR TITLE
Fix - onExit & onSuccess

### PR DESCRIPTION
**Pull request by Arthuzuga**
# Objective

the aim of this PR is to fix a bug caused when existing multi iFrames at the same time and postMessage doesn't know how to handle its events.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,96 +1,115 @@
...
-    handlerMessageListener () {
-        window.addEventListener('message', event => {
-            const { origin, data } = event;
-            const { hook, payload } = data;
-            if(origin !== this.getOrigin()) return;
-            this.payload = payload;
-            this.ee.emit(hook);
-        })
-    };
+ handleMessage(event) {
```
Comment (nic) [inline src/index.ts]: This method is not being used. What's the idea behind adding it to this PR?

**Review by nic — CHANGES_REQUESTED** [review]

File: src/index.ts [hunk 2]
```diff
@@ -1,96 +1,105 @@
...
 
-    onSuccess (callback) {
-        this.ee.on('onsuccess', () => {
-            callback();
-        });
-    };
+ onSuccess(callback) {
+  this.ee.on("onsuccess", () => {
+   callback();
+   const ct = document.querySelector("#connect-embedded");
```
Comment (nic) [inline src/index.ts]:
```suggestion
   const ct = document.querySelector("#connect-embedded-18100062243781");
```

File: src/index.ts [hunk 3]
```diff
@@ -1,96 +1,105 @@
...
 
-    onExit (callback) {
-        this.ee.on('onexit', () => {
-            callback();
-        });
-    };
+ onExit(callback) {
+  this.ee.on("onexit", () => {
+   callback();
+   const ct = document.querySelector("#connect-embedded");
```
Comment (nic) [inline src/index.ts]:
```suggestion
   const ct = document.querySelector("#connect-embedded-18100062243781");
```

**Review by nic — CHANGES_REQUESTED** [review]

**Review by nic — APPROVED** [review]

**Review by gmoura — APPROVED** [review]

**Pull request merged**